maps.xml

```diff
@@ -173,9 +173,9 @@
   <!-- map name="028-2.tmx">
     <atlas name="interiorwoodland"/>
   </map -->
-  <!-- map name="029-2.tmx">
-    <atlas name="interiorwoodland"/>
-  </map -->
+  <map name="029-2.tmx">
+    <atlas name="interiordesert"/>
```

Comment:
Look at the interiorwoodland and interiordesert atlases and you will see that it fits interiordesert more